go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -245,7 +260,28 @@ func (oc *BaseNetworkController) reconcile(netInfo util.NetInfo, setNodeFailed f
 		}
 	}
 
-	return nil
+	namespaceAdded := false
+	for _, ns := range reconcileNamespaces {
+		namespace, err := oc.watchFactory.GetNamespace(ns)
+		if err != nil {
+			klog.Infof("Failed to get namespace %s for reconciling network %s: %v", ns, oc.GetNetworkName(), err)
+			continue
+		}
+		err = oc.retryNamespaces.AddRetryObjWithAddNoBackoff(namespace)
```

Comment:
will have to reconcile only namespaces which match on network policy peer namespace selector, otherwise it adds all namespaces into gress policy ACL, will fix it tomorrow.